Package.swift

```diff
@@ -19,13 +19,15 @@ var targets: [PackageDescription.Target] = [
     .target(name: "NIOCore",
             dependencies: ["NIOConcurrencyHelpers", "CNIOLinux"]),
     .target(name: "_NIODataStructures"),
+    .target(name: "NIOEmbedded", dependencies: ["NIOCore", "_NIODataStructures"]),
```

Comment:
Don't like the name.

This is all about Testing really?  Embedded also don't give much scope for adding things here.  Also makes me think of embedded systems.

Reply:
That’s true, but I don’t want us to push too hard on testing: while this is its primary use, there are reasons you might want to use it in production code.

Maybe `NIOSingleThreaded`?

Reply:
I don't mind the `NIOEmbedded` name.

I wonder if explaining the various "main" NIO modules (`NIO`, `NIOCore`, `NIOEmbedded`, `NIOFoundationCompat` etc) in the README would be useful addition.

Reply:
What's the real reason why it doesn't fit anywhere else?  Is regular EL still in NIO[Posix]?

Reply:
Define "regular EL"?

Reply:
The `EventLoop` protocol is in NIOCore. The SelectableEventLoop is in NIO, and will stay there as it's inherently POSIX-specific.

Reply:
Discussed in a stand-up meeting, agreed to go ahead.